src/main/java/core/basesyntax/service/operator/OperatorImpl.java

```diff
@@ -0,0 +1,61 @@
+package core.basesyntax.service.operator;
+
+import core.basesyntax.dao.FruitShopDao;
+import core.basesyntax.dao.FruitShopDaoImpl;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.Transaction;
+import core.basesyntax.service.fileoperations.FileService;
+import core.basesyntax.service.fileoperations.FileServiceImpl;
+import core.basesyntax.service.transactions.strategy.TransactionStrategy;
+import core.basesyntax.service.transactions.strategy.TransactionStrategyImpl;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class OperatorImpl implements Operator {
+    private static final String CSV_SEPARATOR = ",";
+    private static final String LINE_SEPARATOR = System.lineSeparator();
+    private final TransactionStrategy transactionStrategy = new TransactionStrategyImpl();
+    private final FileService fileService = new FileServiceImpl();
+    private final FruitShopDao dao = new FruitShopDaoImpl();
+
+    public OperatorImpl() {
+    }
+
+    @Override
+    public int openShift() {
```

Comment:
it's not obvious why method openShift() returns int and and what does this number means. I think it's better to make it void or boolean